spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/orm/jpa/HibernateProperties.java

```diff
@@ -121,6 +133,12 @@ else if (!result.containsKey(AvailableSettings.USE_NEW_ID_GENERATOR_MAPPINGS)) {
 		}
 	}
 
+	private void applyArchiveScanner(Map<String, Object> result) {
+		if (!StringUtils.isEmpty(this.archiveScanner)) {
+			result.put(AvailableSettings.SCANNER, this.archiveScanner);
```

Comment:
If the property is not set by the user, we should set it with the disabled scanner implementation.

Reply:
set default value for property as disable scanning